### PR TITLE
Automatically port System.Collections and System.Drawing triple slash comments

### PR DESCRIPTION
**Pull request by carlossanlop**
Found an additional batch of triple slash comments that could be ported from System.Collections and System.Drawing.

### CODE REVIEW

**Comment by carlossanlop** [comment]
Adding area owners for review:
    - System.Collections - @safern 
    - System.Drawing - @safern, @maryamariyan
For language review @mairaw @rpetrusha

File: xml/System.Drawing/Size.xml [hunk 1]
```diff
@@ -561,10 +561,10 @@
...
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Dividend of type <see cref="T:System.Drawing.Size" />.</param>
+        <param name="right">Divisor of type <see cref="T:System.Int32" />.</param>
+        <summary>Divides <see cref="T:System.Drawing.Size" /> by an <see cref="T:System.Int32" /> producing <see cref="T:System.Drawing.Size" />.</summary>
+        <returns>Result of type <see cref="T:System.Drawing.Size" />.</returns>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <returns>A new <see cref="T:System.Drawing.Size" />, which contains the result of dividing <paramref name="left" />'s height by <paramref name="right" /> and <paramref name="left" />'s width by <paramref name="right" />, respectively.</returns>
```

File: xml/System.Drawing/Size.xml [hunk 2]
```diff
@@ -595,10 +595,10 @@
         <Parameter Name="right" Type="System.Single" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Dividend of type <see cref="T:System.Drawing.Size" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="left">The dividend.</param>
```

File: xml/System.Drawing/Size.xml [hunk 3]
```diff
@@ -595,10 +595,10 @@
...
+        <param name="right">Divisor of type <see cref="T:System.Int32" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="right">The divisor.</param>
```

File: xml/System.Drawing/Size.xml [hunk 4]
```diff
@@ -595,10 +595,10 @@
...
+        <summary>Divides <see cref="T:System.Drawing.Size" /> by a <see cref="T:System.Single" /> producing <see cref="T:System.Drawing.SizeF" />.</summary>
+        <returns>Result of type <see cref="T:System.Drawing.SizeF" />.</returns>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <returns>The result of dividing <paramref name="left" />'s width and height by <paramref name="right" />.</returns>
```

File: xml/System.Drawing/Size.xml [hunk 5]
```diff
@@ -595,10 +595,10 @@
         <Parameter Name="right" Type="System.Single" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Dividend of type <see cref="T:System.Drawing.Size" />.</param>
+        <param name="right">Divisor of type <see cref="T:System.Int32" />.</param>
+        <summary>Divides <see cref="T:System.Drawing.Size" /> by a <see cref="T:System.Single" /> producing <see cref="T:System.Drawing.SizeF" />.</summary>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <summary>Divides the specified <see cref="T:System.Drawing.Size" /> by the specified single-precision floating-point number.</summary>
```

File: xml/System.Drawing/Size.xml [hunk 6]
```diff
@@ -817,10 +817,10 @@
         <Parameter Name="right" Type="System.Int32" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Multiplicand of type <see cref="T:System.Drawing.Size" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="left">The multiplicand.</param>
```

File: xml/System.Drawing/Size.xml [hunk 7]
```diff
@@ -817,10 +817,10 @@
...
+        <param name="right">Multiplier of type <see cref="T:System.Int32" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="right">The multiplier.</param>
```

File: xml/System.Drawing/Size.xml [hunk 8]
```diff
@@ -817,10 +817,10 @@
...
+        <summary>Multiplies <see cref="T:System.Drawing.Size" /> by an <see cref="T:System.Int32" /> producing <see cref="T:System.Drawing.Size" />.</summary>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <summary>Multiplies the specified <see cref="T:System.Drawing.Size" /> by the specified integer.</summary>
```

File: xml/System.Drawing/Size.xml [hunk 9]
```diff
@@ -817,10 +817,10 @@
...
+        <returns>Product of type <see cref="T:System.Drawing.Size" />.</returns>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <returns>The result of multiplying <paramref name="left" />'s width and height by <paramref name="right" />.</returns>
```

File: xml/System.Drawing/Size.xml [hunk 10]
```diff
@@ -851,10 +851,10 @@
         <Parameter Name="right" Type="System.Single" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Multiplicand of type <see cref="T:System.Drawing.Size" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="left">The multiplicand.</param>
```

File: xml/System.Drawing/Size.xml [hunk 11]
```diff
@@ -851,10 +851,10 @@
...
+        <param name="right">Multiplier of type <see cref="T:System.Single" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="right">The multiplier.</param>
```

File: xml/System.Drawing/Size.xml [hunk 12]
```diff
@@ -851,10 +851,10 @@
...
+        <summary>Multiplies <see cref="T:System.Drawing.Size" /> by a <see cref="T:System.Single" /> producing <see cref="T:System.Drawing.SizeF" />.</summary>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <summary>Multiplies the specified <see cref="T:System.Drawing.Size" /> by the specified single-precision floating-point number.</summary>
```

File: xml/System.Drawing/Size.xml [hunk 13]
```diff
@@ -851,10 +851,10 @@
...
+        <returns>Product of type <see cref="T:System.Drawing.SizeF" />.</returns>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <returns>The result of multiplying <paramref name="left" />'s width and height by <paramref name="right" />.</returns>
```

File: xml/System.Drawing/Size.xml [hunk 14]
```diff
@@ -885,10 +885,10 @@
         <Parameter Name="right" Type="System.Drawing.Size" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Multiplier of type <see cref="T:System.Int32" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="left">The multiplier.</param>
```

File: xml/System.Drawing/Size.xml [hunk 15]
```diff
@@ -885,10 +885,10 @@
...
+        <param name="right">Multiplicand of type <see cref="T:System.Drawing.Size" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="right">The multiplicand.</param>
```

File: xml/System.Drawing/Size.xml [hunk 16]
```diff
@@ -885,10 +885,10 @@
...
+        <summary>Multiplies a <see cref="T:System.Drawing.Size" /> by an <see cref="T:System.Int32" /> producing <see cref="T:System.Drawing.Size" />.</summary>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <summary>Multiplies the specified integer by the specified <see cref="T:System.Drawing.Size" />.</summary>
```

File: xml/System.Drawing/Size.xml [hunk 17]
```diff
@@ -885,10 +885,10 @@
...
+        <returns>Product of type <see cref="T:System.Drawing.Size" />.</returns>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <returns>The result of multiplying <paramref name="right" />'s width and height by <paramref name="left" />.</returns>
```

File: xml/System.Drawing/Size.xml [hunk 18]
```diff
@@ -919,10 +919,10 @@
         <Parameter Name="right" Type="System.Drawing.Size" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Multiplier of type <see cref="T:System.Single" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="left">The multiplier.</param>
```

File: xml/System.Drawing/Size.xml [hunk 19]
```diff
@@ -919,10 +919,10 @@
...
+        <param name="right">Multiplicand of type <see cref="T:System.Drawing.Size" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <param name="right">The multiplicand.</param>
```

File: xml/System.Drawing/Size.xml [hunk 20]
```diff
@@ -919,10 +919,10 @@
...
+        <summary>Multiplies <see cref="T:System.Drawing.Size" /> by a <see cref="T:System.Single" /> producing <see cref="T:System.Drawing.SizeF" />.</summary>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <summary>Multiplies the specified single-precision floating-point number by the specified <see cref="T:System.Drawing.Size" />.</summary>
```

File: xml/System.Drawing/Size.xml [hunk 21]
```diff
@@ -919,10 +919,10 @@
...
+        <returns>Product of type <see cref="T:System.Drawing.SizeF" />.</returns>
```
Comment (mairaw) [inline xml/System.Drawing/Size.xml]:
```suggestion
        <returns>The result of multiplying <paramref name="right" />'s width and height by <paramref name="left" />.</returns>
```

File: xml/System.Drawing/SizeF.xml [hunk 22]
```diff
@@ -574,10 +574,10 @@
         <Parameter Name="right" Type="System.Single" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Dividend of type <see cref="T:System.Drawing.SizeF" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <param name="left">The dividend.</param>
```

File: xml/System.Drawing/SizeF.xml [hunk 23]
```diff
@@ -574,10 +574,10 @@
...
+        <param name="right">Divisor of type <see cref="T:System.Int32" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <param name="right">The divisor.</param>
```

File: xml/System.Drawing/SizeF.xml [hunk 24]
```diff
@@ -574,10 +574,10 @@
...
+        <summary>Divides <see cref="T:System.Drawing.SizeF" /> by a <see cref="T:System.Single" /> producing <see cref="T:System.Drawing.SizeF" />.</summary>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <summary>Divides the specified <see cref="T:System.Drawing.SizeF" /> by the specified single-precision floating-point number.</summary>
```

File: xml/System.Drawing/SizeF.xml [hunk 25]
```diff
@@ -574,10 +574,10 @@
...
+        <returns>Result of type <see cref="T:System.Drawing.SizeF" />.</returns>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <returns>The result of dividing <paramref name="left" />'s width and height by <paramref name="right" />.</returns>
```

File: xml/System.Drawing/SizeF.xml [hunk 26]
```diff
@@ -728,10 +728,10 @@
         <Parameter Name="right" Type="System.Single" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Multiplicand of type <see cref="T:System.Drawing.SizeF" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <param name="left">The multiplicand.</param>
```

File: xml/System.Drawing/SizeF.xml [hunk 27]
```diff
@@ -728,10 +728,10 @@
...
+        <param name="right">Multiplier of type <see cref="T:System.Single" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <param name="right">The multiplier.</param>
```

File: xml/System.Drawing/SizeF.xml [hunk 28]
```diff
@@ -728,10 +728,10 @@
...
+        <summary>Multiplies <see cref="T:System.Drawing.SizeF" /> by a <see cref="T:System.Single" /> producing <see cref="T:System.Drawing.SizeF" />.</summary>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <summary>Multiplies the specified <see cref="T:System.Drawing.SizeF" /> by the specified single-precision floating-point number.</summary>
```

File: xml/System.Drawing/SizeF.xml [hunk 29]
```diff
@@ -728,10 +728,10 @@
...
+        <returns>Product of type <see cref="T:System.Drawing.SizeF" />.</returns>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <returns>The result of multiplying <paramref name="left" />'s width and height by <paramref name="right" />.</returns>
```

File: xml/System.Drawing/SizeF.xml [hunk 30]
```diff
@@ -762,10 +762,10 @@
         <Parameter Name="right" Type="System.Drawing.SizeF" Index="1" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="left">To be added.</param>
-        <param name="right">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="left">Multiplier of type <see cref="T:System.Single" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <param name="left">The multiplier.</param>
```

File: xml/System.Drawing/SizeF.xml [hunk 31]
```diff
@@ -762,10 +762,10 @@
...
+        <param name="right">Multiplicand of type <see cref="T:System.Drawing.SizeF" />.</param>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <param name="right">The multiplicand.</param>
```

File: xml/System.Drawing/SizeF.xml [hunk 32]
```diff
@@ -762,10 +762,10 @@
...
+        <summary>Multiplies <see cref="T:System.Drawing.SizeF" /> by a <see cref="T:System.Single" /> producing <see cref="T:System.Drawing.SizeF" />.</summary>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <summary>Multiplies the specified single-precision floating-point number by the specified <see cref="T:System.Drawing.SizeF" />.</summary>
```

File: xml/System.Drawing/SizeF.xml [hunk 33]
```diff
@@ -762,10 +762,10 @@
...
+        <returns>Product of type <see cref="T:System.Drawing.SizeF" />.</returns>
```
Comment (mairaw) [inline xml/System.Drawing/SizeF.xml]:
```suggestion
        <returns>The result of multiplying <paramref name="right" />'s width and height by <paramref name="left" />.</returns>
```

**Comment by carlossanlop** [comment]
@mairaw thanks for merging your suggestions. Is this good to go or should we address something else?
@safern @maryamariyan do you have any additional suggestions we should address before merging?

**Comment by carlossanlop** [comment]
@rpetrusha while Maira is away, could you please merge this one?

**Pull request merged**